libs/openFrameworks/graphics/ofImage.cpp

```diff
@@ -358,12 +375,12 @@ static bool saveImage(const ofPixels_<PixelType> & _pix, const of::filesystem::p
 	}
 
 	ofFilePath::createEnclosingDirectory(_fileName);
-	auto fileName = ofToDataPath(_fileName);
+	auto fileName = ofToDataPathFS(_fileName).string().c_str();
 	FREE_IMAGE_FORMAT fif = FIF_UNKNOWN;
-	fif = FreeImage_GetFileType(fileName.c_str(), 0);
+	fif = FreeImage_GetFileType(fileName, 0);
 	if(fif == FIF_UNKNOWN) {
 		// or guess via filename
-		fif = FreeImage_GetFIFFromFilename(fileName.c_str());
+		fif = FreeImage_GetFIFFromFilename(fileName);
```

Comment:
there is a `FreeImage_GetFIFFromFilenameU()`, why not maintain the `#ifdef windows` pattern? instead of forcing the `fileName` into a `.string()` (again risking data loss/exception)
```
fif = FreeImage_GetFIFFromFilenameU(_fileName().c_str()); // works on windows
```
(same applies below to other "U" suffixed-functions)

Reply:
OK I think this one is resolved now